execution/engine/emr_engine.go

```diff
@@ -107,22 +107,32 @@ func (emr *EMRExecutionEngine) Execute(executable state.Executable, run state.Ru
 }
 
 func (emr *EMRExecutionEngine) generateApplicationConf(executable state.Executable, run state.Run, manager state.Manager) []*emrcontainers.Configuration {
-	properties := map[string]*string{
+	sparkDefaults := map[string]*string{
 		"spark.kubernetes.driver.podTemplateFile":   emr.driverPodTemplate(executable, run, manager),
 		"spark.kubernetes.executor.podTemplateFile": emr.executorPodTemplate(executable, run, manager),
 		"spark.kubernetes.container.image":          &run.Image,
 		"spark.eventLog.dir":                        aws.String(fmt.Sprintf("s3a://%s/%s", emr.s3LogsBucket, emr.s3EventLogPath)),
 		"spark.history.fs.logDirectory":             aws.String(fmt.Sprintf("s3a://%s/%s", emr.s3LogsBucket, emr.s3EventLogPath)),
 		"spark.eventLog.enabled":                    aws.String(fmt.Sprintf("true")),
 	}
+	hiveDefaults := map[string]*string{}
 
 	for _, k := range run.SparkExtension.ApplicationConf {
-		properties[*k.Name] = k.Value
+		sparkDefaults[*k.Name] = k.Value
 	}
+
+	for _, k := range run.SparkExtension.HiveConf {
```

Comment:
Is this pulling AWS's default HiveConf? And our overrides in the operator should be able to replace these?

Reply:
no defaults here.